x-pack/solutions/observability/plugins/infra/public/components/asset_details/template/flyout.tsx

```diff
@@ -49,6 +50,10 @@ export const Flyout = ({
       onClose={handleOnClose}
       data-component-name={ASSET_DETAILS_FLYOUT_COMPONENT_NAME}
       data-asset-type={asset.type}
+      aria-labelledby={i18n.translate('xpack.infra.assetDetails.flyout.ariaLabel', {
```

Comment:
Is this right `aria-labelledby`?

Reply:
Yes, https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-labelledby
It's best used for dialogs (modals, flyout, etc)